erpnext/hr/doctype/attendance_request/attendance_request.py

```diff
@@ -54,6 +54,12 @@ def update_attendance(self):
 							'late_entry': 0,
 							'previous_late_entry': existing_doc.late_entry,
 						})
+					
+					if self.remove_early_exit:
+						changes.update({
+							'early_exit': 0,
+							'previous_early_exit': existing_doc.early_exit,
+						})
```

Comment:
Need to handle Attendance Request on_cancel as well to reset `early_exit` to previous_early_exit`